src/containers/LeaderboardPage/index.js

```diff
@@ -35,19 +35,21 @@ const LeaderboardPage = props => {
   };
 
   return (
-    <div className={`${styles.leaderboardPage}`}>
-      <div className={`${styles.leaderboardHeader}`}>
-        <div className={`${styles.leaderboardHeaderBody}`}>
-          <Text style="H1" color="white">
-            Leaderboard
-          </Text>
+    <div className={`${styles.leaderboardPage} col-xs`}>
```

Comment:
I'm confused here. Why does it start off with a col-xs?  Usually a column will have a number on it and will preface a row.